### DISCUSSION TITLE
Database Connection Error: 28P01 undefined:undefined

### DISCUSSION THREAD

**Discussion by salehmotiwala** [requarks/wiki#3060] [Help / Questions]
I'm installing Wiki.js on windows 10. 
**OS: Windows 10
Wiki.js version: 2.5.170
Database engine: PostgreSQL 13.1**
I'm following this guide: https://docs.requarks.io/install/windows 
However, I keep getting this **error: Database Connection Error: 28P01 undefined:undefined
error: Database Initialization Error: password authentication failed for user "PostgreSQL"**

**Comment by aelson** [comment]
Same error here. I'm on Ubuntu, installing with minikube and helm.

**Reply by salehmotiwala** [reply]
I wasn't adding the password right. I have it figured out now

**Comment by theAkito** [comment]
Still happens, today. One pod is already successfully connected, but the second pod on `helm upgrade` cannot connect. Therefore, the password setup must be correct or else the first pod wouldn't have connected in the first place....

**Comment by theAkito** [comment]
Ran into this again, today, just to find that I've been here, already.

All I did was restarting the deployment & now the resulting pod cannot connect. I even double-checked the secret references etc. They are all utterly correct. This is annoying.

**Reply by theAkito** [reply]
"Fixed" it by setting a new password everywhere, including the database itself & all resource definitions...
